src/components/SearchBar.tsx

```diff
@@ -140,7 +139,6 @@ export class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
         this.searchInputField.value = searchValueCopy.join("");
         this.searchInputField.focus();
         this.searchInputField.setSelectionRange(cursor, cursor);
-        this.props.onTextChanged(this.searchInputField.value);
```

Comment:
But wouldn't you want to update the value (search term) after the forward delete action?